homeassistant/components/apsystems/entity.py

```diff
@@ -0,0 +1,33 @@
+"""APsystems base entity."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import ApsystemsConfigEntry, ApSystemsData
+from .const import DOMAIN
+from .coordinator import ApSystemsDataCoordinator
+
+
+class ApsystemsEntity(CoordinatorEntity[ApSystemsDataCoordinator]):
```

Comment:
I think the biggest thing to note is that the other entities aren't going to be attached to the coordinator and thus should not be a coordinator entity, since that would stop the polling.

Instead this could inherit Entity, and then the sensor entity would inherit both the ApsystemsEntity, the CoordinatorEntity and the sensor entity.